toolkit/templates/summary-table.html

```diff
@@ -279,7 +279,7 @@ <h2 class="summary-item-heading"{% if id %} id="{{ id }}"{% endif %}>
 
 
 {% macro multiquestion(value, assurance) -%}
-  {% call field() -%}
+  {% call field(False, False, False, False, False, False, False, '', False, 'nowrap') -%}
```

Comment:
You should be able to do this instead:
{% call field(className='nowrap') %}

Also, the argument isn't *the* class name; it's an optional extra class.  Maybe extra_classes is a better name.  (Note that the other macro arguments use underscored_names not camelCase.)